README.md

```diff
@@ -85,6 +87,8 @@ To see a list of all of your available iOS devices, use `xcrun simctl list devic
 
 ### Troubleshooting
 
+If the app doesn't run on your first attempt, it may help to double-check the set up of your development environment against the latest requirements in [React Native's documentation](https://reactnative.dev/docs/environment-setup). With Android Studio, for example, you will need to configure the ANDROID_HOME environment variable and ensure that your version of JDK matches the latest requirements.
```

Comment:
> If the app doesn't run on your first attempt

👋 @SiobhyB ! This feels too "open" to me and judging from your [comment here](https://github.com/wordpress-mobile/gutenberg-mobile/pull/3063#issuecomment-769698986), it might not even be an app issue but a development environment issue (say, if the Android emulator itself doesn't work).

What do you think about adding specific examples so the context is narrower? Something like this for example:
`If the Android emulator doesn't start correctly, or compiling fails with 'Could not initialize class org.codehaus.groovy.runtime.InvokerHelper' or similar, it may help to double-check...`

Reply:
Thank you for your feedback @hypest! I did lean towards keeping it a bit more open for the reasons you touched on in your last comment, I get the sense that there are many, many possible setup errors that can't all be covered in detail in the docs. I really like your suggestion, though, as it's more specific but still broad enough. I've updated. :)